driver/mysql/mysql_test.go

```diff
@@ -29,6 +29,22 @@ func TestMigrate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	migrate(t, driverUrl)
+
+	if _, err := connection.Exec(`DROP TABLE IF EXISTS yolo, yolo1, ` + tableName); err != nil {
```

Comment:
This drops the tables used in the previous `migrate` test so that we can create a version of the migration table using the old 32-bit integers and without the yolo table so the migrations all work the same.